internal/fs/tarfs/tarfs_test.go

```diff
@@ -268,3 +268,28 @@ func TestTarFS_Stat_Unsupported(t *testing.T) {
 		}
 	}
 }
+
+func TestTarFS_New_NonTarFile(t *testing.T) {
+
+	emptyfile := "testdata/emptyfile.tar"
+	nonemptyfile := "testdata/nonempty.tar"
+	tarwithoutextension := "testdata/tarwithoutextension"
+
+	_, err := New(emptyfile)
+
```

Comment:
nit: I think it's more readable without this empty line.

Reply:
Removed the line.